Makefile

```
@@ -0,0 +1,19 @@
+.PHONY: db-up db-down db-clean
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)